# Request
---
# Response
int16[6] tempvalue
bool success
